ArduCopter/ReleaseNotes.txt

```diff
@@ -1,5 +1,83 @@
 ArduPilot Copter Release Notes:
 ------------------------------------------------------------------
+Copter 4.1.0-beta1 14-Apr-2021
+Changes from 4.0.7
+1) EKF changes:
+    a) EKF3 is default estimator (EKF2 is available as an option)
+    b) External AHRS/IMU support (e.g. VectorNav)
+    c) Gaussian Sum Filter (GSF) allows emergency yaw correction using GPS
+    d) GPS-for-yaw (dual F9 UBlox GPS can provide yaw)
+    e) Lane switching logic improvements
+    f) Sensor affinity (improves sensor failure redundancy)
+    g) Source switching for GPS/Non-GPS transitions (see EK3_SRCx_ parameters)
+    h) Yaw estimation and reliability improvements
+2) Control and Navigation improvements:
+    a) SCurves for waypoint navigation
+    b) Auto mdoe arming and takeoff (see AUTO_OPTIONS)
```

Comment:
b) Auto mode arming and takeoff (see AUTO_OPTIONS)

Reply:
Add logging for 2D PID objects
Add Feed Forward to 2D PID object
Add slew limiting to AC_PID object
Update Position Controller

Reply:
Added yaw imbalance check (https://github.com/ArduPilot/ardupilot/pull/15744)